### PR TITLE
Add document for tiup-cluster

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

The translation of `使用中控机系统自带的 SSH 客户端连接集群` is from https://docs.pingcap.com/tidb-data-migration/dev/maintain-dm-using-tiup#use-the-systems-native-ssh-client-to-connect-to-cluster

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/4066
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: tiup/tiup-cluster.md [hunk 1]
```diff
@@ -626,4 +634,35 @@ The TiUP data is stored in the `.tiup` directory in the user's home directory. T
 
 > **Note:**
 >
-> It is recommended that you back up the `.tiup` directory regularly to avoid the loss of TiUP data caused by abnormal conditions, such as disk damage of the control machine. 
+> It is recommended that you back up the `.tiup` directory regularly to avoid the loss of TiUP data caused by abnormal conditions, such as disk damage of the control machine.
+
+## Use the system's native SSH client to connect to cluster
```
Comment (ran-huang) [inline tiup/tiup-cluster.md:639]: This section should be under L623. 🤔

File: tiup/tiup-cluster.md [hunk 2]
```diff
@@ -626,4 +634,35 @@ The TiUP data is stored in the `.tiup` directory in the user's home directory. T
...
+- Upgrade a cluster: `tiup cluster upgrade ... --native-ssh`
+
+You can add `--native-ssh` in all cluster operation commands above to use the system's native SSH client.
+
+To avoid adding such a flag in every command, you can use the `TIUP_NATIVE_SSH` system variable to specify whether to use the local SSH client:
+
+```sh
+export TIUP_NATIVE_SSH=true
+# or
+export TIUP_NATIVE_SSh=1
```
Comment (ran-huang) [inline tiup/tiup-cluster.md:659]:
```suggestion
export TIUP_NATIVE_SSH=1
```
@lucklove Is this a typo?

File: tiup/tiup-cluster.md (hunk 1 above)
Comment (TomShawn) [reply tiup/tiup-cluster.md:624]: Moved

File: tiup/tiup-cluster.md (hunk 2 above)
Comment (lucklove) [reply tiup/tiup-cluster.md:644]: YES

**Review by ran-huang — APPROVED** [review]

**Pull request merged**